### PR TITLE
3.x fair share scheduler documetation

### PR DESCRIPTION
**Pull request by nickva**
A short description on how the algorithm works along with the configuration sections.

Main PR: https://github.com/apache/couchdb/pull/3364

### CODE REVIEW

File: src/config/replicator.rst [hunk 1]
```diff
@@ -249,3 +249,52 @@ Replicator Database Configuration
...
+.. config:section:: replicator.shares :: Per-Database Fair Share Allocation
+
+    .. config:option:: $replicator_db
+
+        .. versionadded:: 3.2.0
+
+        Fair share configuration section. More shares result in a
+        higher chance that jobs from that db get to run. The default
+        value is 100, minimum is 1 and maximum is 1000. The
+        configuration may be set even if the database does not exit::
```
Comment (bessbd) [inline src/config/replicator.rst:296]: Nit
```suggestion
        configuration may be set even if the database does not exist::
```

File: src/replication/replicator.rst [hunk 2]
```diff
@@ -21,6 +21,11 @@ Replicator Database
    anymore. There are new replication job states and new API endpoints
    ``_scheduler/jobs`` and ``_scheduler/docs``.
 
+.. versionchanged:: 3.2.0 Fair share scheduling was
+   introduced. Multiple ``_replicator`` databases get an equal chance
```
Comment (bessbd) [inline src/replication/replicator.rst:25]: Is the line wrap important here? Should `introduced` go to the line above?

**Review by bessbd — COMMENTED** [review]
I've only skimmed through the change, but it looks good to me in general.

File: src/config/replicator.rst (hunk 1 above)
Comment (nickva) [reply src/config/replicator.rst]: Good eye! Thanks. Pushed the fix

File: src/replication/replicator.rst (hunk 2 above)
Comment (nickva) [reply src/replication/replicator.rst]: You're right. It should go in the line above. Updated.

**Review by bessbd — APPROVED** [review]
LGTM, +1. Please mind that I did not generate the docs locally, so I don't have info of how it actually "looks".

**Pull request merged**